imports/sound-handler/sound-handler.ts

```diff
@@ -0,0 +1,104 @@
+import { DeepClient } from "@deep-foundation/deeplinks/imports/client";
+import speech from "@google-cloud/speech"
+
+
+export default async function insertSoundHandler(deep: DeepClient) {
+  const fileTypeLinkId = await deep.id("@deep-foundation/core", "SyncTextFile");
+  const containTypeLinkId = await deep.id("@deep-foundation/core", "Contain");
+  const supportsId = await deep.id("@deep-foundation/core", "dockerSupportsJs");
+  const handlerTypeLinkId = await deep.id("@deep-foundation/core", "Handler");
+  const packageId = await deep.id("@deep-foundation/sound-handler");
+  const handleOperationTypeLinkId = await deep.id("@deep-foundation/core", "HandleInsert")
+  const triggerTypeLinkId = await deep.id("@deep-foundation/audiorecord", "AudioChunk")
```

Comment:
This package must not depend on @deep-foundation/audiorecord. Handler should handle insert event of a link which type is contained in this package, not another.

Example: you insert a link of type Transcribe from a user to a link with type Sound from sound package